l10n/en/firefox/features/private-browsing-2023.ftl

```diff
@@ -14,6 +14,7 @@ features-private-browsing-firefox-protects = { -brand-name-firefox } protects yo
 
 features-private-browsing-if-you-share-a-computer = If you share a computer with other people or if you want to limit how much data websites can collect about you, you can use private browsing mode in { -brand-name-firefox }. Private browsing erases the digital tracks you leave behind when you browse online — think of them like footprints through the woods.
 features-private-browsing-what-does-private-browsing-do = What does private browsing do?
+features-private-browsing-private-browsing-mode-opens-v2 = Private browsing mode opens a new browser window. When you close the last private browsing window, your browsing history and any tracking cookies from websites you visited will be erased. <strong>{ -brand-name-firefox } Pro Tip:</strong> Don’t forget to close all your private browsing windows when you’re done!
 features-private-browsing-private-browsing-mode-opens = Private browsing mode opens a new browser window. When you close it, your browsing history for that window and any tracking cookies from websites you visited will be erased. <strong>{ -brand-name-firefox } Pro Tip:</strong> Don’t forget to close the private browsing window when you’re done!
```

Comment:
The previous string should have been marked as obsolete (comment with date for removal).

Reply:
My bad - I'm not in the habit of fluent changes. Will add that now as a separate PR and tag you for R

Reply:
Please tag @peiying2